packages/subiquity_client/lib/test/test_server.dart

```diff
@@ -42,7 +42,8 @@ class TestServer {
     return socketPath;
   }
 
-  void stop() {
+  Future<void> stop() async {
     _serverProcess.kill();
```

Comment:
Pass `ProcessSignal.sigkill` to make sure it dies? :)

Reply:
Thanks for the suggestions, not sure yet what to do here. The subiquity server appears to freeze now and then upon receiving an invalid function parameter. Trying to figure out if it's Dart-side or Python-side at the moment. I may just disable that test for now...